src/main/res/xml/prefs_behaviour.xml

```diff
@@ -130,9 +130,11 @@
 
     <PreferenceCategory android:title="@string/pref_behaviour_posts_header">
 
-        <CheckBoxPreference android:title="@string/pref_behaviour_nsfw_title"
-                            android:key="@string/pref_behaviour_nsfw_key"
-                            android:defaultValue="false"/>
+        <ListPreference android:title="@string/pref_behaviour_nsfw_title"
+						android:key="@string/pref_behaviour_nsfw_key"
```

Comment:
When altering a preference, a different key should be used to the original (I'd go for `@string/pref_behaviour_nsfw_2_key` which should be added to strings.xml with the text `pref_behaviour_nsfw_2_key`).